application/frontend/src/components/LoginForm.tsx

```diff
@@ -1,13 +1,34 @@
-import { Form, Input, Button, Checkbox } from "antd";
-import { Link } from "react-router-dom";
+import { Form, Input, Button } from "antd";
+import axios from "axios";
+import { Link, useHistory } from "react-router-dom";
 
 export const LoginForm = () => {
+  const history = useHistory();
+
   const onFinish = (values: any) => {
-    console.log("Success:", values);
+    axios
+      .post("/login", {
+        email: values.email,
```

Comment:
No need to email to login. @veyisTurgut

Reply:
Username or email, one of them is needed. Which one does not matter.